database/migrations/2023_02_19_143928_create_cart_items_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('cart_items', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedInteger('qty');
+            $table->string('name');
+            $table->unsignedInteger('price');
+            $table->foreignId('cart_id');
+            $table->foreignId('product_id');
```

Comment:
refrence and nullOnDelete?

Reply:
If a product removed then cart's item that related to this product must be removed too. it's true?
too
if a cart removed then cart's items that related to this cart must be removed too. it's true?